CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+## 2.17.1 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.2.1890**
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented support for ``enable_monitor_gateway_subnets`` and ``monitor_exclude_list`` in **aviatrix_gateway**
+2. Implemented support for ``idle_timeout`` and ``renegotiation_interval`` in **aviatrix_gateway**
+3. Implemented support for ``enable_active_standby`` in **aviatrix_transit_gateway**
+4. Implemented support for ``switch_to_ha_standby_gateway`` in **resource_aviatrix_transit_external_device_conn**
```

Comment:
This could be written as a bullet point with sub points.
3. Implemented [Active-Standby](https://docs.aviatrix.com/HowTos/transit_advanced.html#active-standby) support for Transit Network workflows:
- `enable_active_standby` in **aviatrix_transit_gateway**
- `switch_to_ha_standby_gateway` in **aviatrix_transit_external_device_conn**